### PR TITLE
New package: diffuse-0.5.0beta.1

### PR DESCRIPTION
**Pull request by dimaburmistrov**

### CODE REVIEW

File: srcpkgs/diffuse/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'diffuse'
+pkgname=diffuse
+version=0.4.8
+revision=1
+archs=noarch
+build_style=fetch
```
Comment (abenson) [inline srcpkgs/diffuse/template:6]: Probably want to omit `build_style` altogether in this case. `fetch` will download the file but not extract it.

File: srcpkgs/diffuse/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+depends="pygtk python"
```
Comment (abenson) [inline srcpkgs/diffuse/template:7]: We are no longer accepting python2 packages. Is there any chance of getting it to work with python3?

Comment (dimaburmistrov) [reply srcpkgs/diffuse/template:7]: It seems like the author has stopped working on this project few years ago. I'm also not familiar with projects code, so can't say anything about porting to py3.

What do you think of this situation?

Comment (abenson) [reply srcpkgs/diffuse/template:6]: python2 and abandoned projects are both blockers for inclusion

Comment (dimaburmistrov) [reply srcpkgs/diffuse/template:6]: Okay. Completely understand.

I've found fork of this tool (with py3 & gtk3) - made a template for latest release, but it has annoying bug (fixed in later commits).

I asked for spawning new release (with bugfix) [here](https://github.com/MightyCreak/diffuse/issues/42) - it's still open.

I can't say how long would it take to bring that fork into the void-packages, so what is the best way here? - Close this PR and create new one later? Or leave this open until fresh release will be spawned in that repo (py3-fork)?

Comment (ericonr) [reply srcpkgs/diffuse/template:6]: If the fix is tiny (one can hope), you can create a patch and package their latest release. Try to ask them about it, perhaps. That way they don't have to cut a release.

Comment (dimaburmistrov) [reply srcpkgs/diffuse/template:6]: Okay. I'll play with the "patch" approach on holidays and come back with some suitable solution. Thanks.

File: srcpkgs/diffuse/template (hunk 1 above)
Comment (dimaburmistrov) [reply srcpkgs/diffuse/template]: Fixed. Thank you.

File: srcpkgs/diffuse/template (hunk 2 above)
Comment (dimaburmistrov) [reply srcpkgs/diffuse/template]: Updated PR with Py3-version of this application (also changed the title). Used fresh release (with bug-fixes). It works on my machine.

I hope now it's package the proper way. What would you say? - Can we merge it?

File: srcpkgs/diffuse/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'diffuse'
+pkgname=diffuse
+_dist_ver=0.5.0
+_dist_rel=beta.1
+version="${_dist_ver}${_dist_rel}"
+revision=1
+archs=noarch
```
Comment (ericonr) [inline srcpkgs/diffuse/template:7]: noarch is no longer used.

File: srcpkgs/diffuse/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+wrksrc="${pkgname}-${_dist_ver}-${_dist_rel}"
+depends="python3-gobject"
+short_desc="Graphical tool for merging and comparing text files (Python3-fork)"
```
Comment (ericonr) [inline srcpkgs/diffuse/template:10]: I don't think specifying that it is a fork is necessary.

File: srcpkgs/diffuse/template [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="Dima Burmistrov <dmabrm@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/MightyCreak/diffuse"
+distfiles="https://github.com/MightyCreak/diffuse/archive/v${_dist_ver}-${_dist_rel}.tar.gz"
+checksum=00f62302e0bad049b99dbc8f9c12b2c6eb10046489f52a11069dcb326d3f41fb
+python_version=3
+
+do_install() {
+	cp -av src/* "${DESTDIR}/"
```
Comment (ericonr) [inline srcpkgs/diffuse/template:19]: There is a 0.6.0 release, and it requires a bit more complicated setup than this. I believe `build_style=meson` will be necessary.

**Pull request closed without merging**